src/display/node_stream.js

```diff
@@ -304,11 +304,9 @@ class PDFNodeStreamFullReader extends BaseFullReader {
           disableRange: this._disableRange,
         });
 
-      if (allowRangeRequests) {
-        this._isRangeSupported = true;
-      }
+      this._isRangeSupported = allowRangeRequests;
```

Comment:
This change should also be done in `src/display/network.js` above.

Reply:
That's not strictly necessary, since there's a subtle difference between `network.js` respectively `fetch_stream.js`/`node_stream.js` regarding the initialization of the properties.
In `network.js` everything is explicitly disabled until we've determined that support exist, however in e.g. `fetch_stream.js` you could probably assume that stream support exists (by design). This is a bit counter-intuitive at first, but I'm not sure if we want to change that particular instance!?

For `network.js`, note these definitions https://github.com/mozilla/pdf.js/blob/9767b8a9f1ae7cf5250da648deb62b5f1c4d7292/src/display/network.js#L337-L338 and then how there're explicitly set in https://github.com/mozilla/pdf.js/blob/9767b8a9f1ae7cf5250da648deb62b5f1c4d7292/src/display/network.js#L370-L380
Now compare with e.g. `fetch_stream.js`, where these definitions are used https://github.com/mozilla/pdf.js/blob/9767b8a9f1ae7cf5250da648deb62b5f1c4d7292/src/display/fetch_stream.js#L83-L84 and then https://github.com/mozilla/pdf.js/blob/9767b8a9f1ae7cf5250da648deb62b5f1c4d7292/src/display/fetch_stream.js#L116-L123.

Reply:
I see. Let's indeed not change that now.